package.json

```diff
@@ -23,6 +23,7 @@
     "@ng-bootstrap/ng-bootstrap": "^3.0.0",
     "angular-font-awesome": "^3.1.2",
     "bootstrap": "^4.1.3",
+    "popper.js": "^1.14.3",
```

Comment:
👍

Reply:
Do we need the donut, it still does not look smooth on the edges.